acceptance/setup/pre_suite/15_setup_repos.rb

```diff
@@ -1,10 +1,24 @@
 def initialize_repo_on_host(host, os)
   case os
   when :debian
-    on host, "wget http://apt.puppetlabs.com/puppetlabs-release-$(lsb_release -sc).deb"
+  if options[:type] == 'aio' then
```

Comment:
nitpick: indent this block here inside a `when` block. Should line up with the last `on` in the block.